pages.ja/common/git.md

```diff
@@ -0,0 +1,30 @@
+# git
+
+> 分散型バージョン管理システム
+> いくつかのサブコマンドがあります。例えば `commit`, `add`, `branch`, `checkout`, `push`, などです。
+> これらには使用方法についての独自のドキュメントがあり、 `tldr git subcommand` で見ることができます。
+> 詳しくはこちら: <https://git-scm.com/>.
+
+- Gitのバージョンを確認する:
+
+`git --version`
+
+- Git全体のヘルプを見る:
+
+`git --help`
+
+- Gitのサブコマンドのヘルプを見る (例えば `clone`, `add`, `push`, `log`, など):
+
+`git help {{subcommand}}`
+
+- Gitのサブコマンドを実行する:
+
+`git {{subcommand}}`
+
+- Gitのサブコマンドを、任意のリポジトリのルートパスを指定して実行する:
+
+`git -C {{path/to/repo}} {{subcommand}}`
```

Comment:
```suggestion
`git -C {{リポジトリ パス}} {{サブコマンド}}`
```

Reply:
I feel the original expression `path/to/repo` is very easy to understand that it cleary means any another path to directory because it contains `/`, so I suggest following expression:

```suggestion
`git -C {{対象とする/リポジトリへの/パス}} {{サブコマンド}}`
```